BUILD.bazel

```diff
@@ -38,12 +41,10 @@ cc_binary(
 cc_test(
     name = "unittest-gcc",
     srcs = glob([
-            "tests/*.h",
-            "tests/*.cpp",
-            "include/sonic/*",
-            "include/sonic/**/*",
-        ]),
-    deps = [ 
+        "tests/*.cpp",
+    ]),
+    deps = [
+        ":sonic-cpp",
```

Comment:
bazel may not report any werrors in the hdrs of :sonic-cpp

Reply:
ok, I'll change back.